_docs/_latest/setup/forseti-on-gke.md

```diff
@@ -123,27 +124,60 @@ module "forseti-on-gke-with-config-validator" {
     # SSH Git repository location, usually in the following
     # format: git@repo-host:repo-owner/repo-name.git
     policy_library_repository_url = ""
+
+    # Whether or not to enable the periodic git-sync.
+    policy_library_sync_enabled   = true
 }
 ```
+#### Policy Library in a GCS bucket
+```bash
+module "forseti-on-gke-with-config-validator" {
+    # Other parameters/variables removed for brevity
 
-#### With Helm
+    # Enable config-validator
+    config_validator_enabled = true
+}
+```
 
+### With Helm
+#### Policy Library in a Git Repository
 In the Helm example above, the following variables are required in the user defined *values.yaml* file.
 
 ```yaml
-# configValidator sets whether or not to deploy config-validator
-configValidator: true
+# configValidator.enabled sets whether or not to deploy config-validator
+configValidator.enabled: true
 
-# gitSyncPrivateSSHKey is the private OpenSSH key generated to allow the git-sync to clone the policy library repository.
-gitSyncPrivateSSHKey: ""
+# configValidator.policyLibrary.gitSync.privateSSHKey
+# is the private OpenSSH key generated to allow the 
+# git-sync to clone the policy library repository.
+configValidator.policyLibrary.gitSync.privateSSHKey: ""
```

Comment:
I believe there is an issue with the recent GCS changes made to the Terraform and Helm. In doing some testing of a new feature, I am having a hard time getting the helm release to successfully deploy the cluster (mainly CV pod will not start). Is there an issue here where Terraform is still providing the `configValidator` prefix to these values when the CV values has been moved into it's own chart? https://github.com/forseti-security/helm-charts/blob/master/charts/config-validator/values.yaml